lib/python/qmk/cli/cformat.py

```diff
@@ -43,7 +43,7 @@ def cformat(cli):
     files = []
     # Core directories for formatting
     core_dirs = ['drivers', 'quantum', 'tests', 'tmk_core']
-    ignores = ['tmk_core/protocol/usb_hid', 'quantum/template']
+    ignores = ['tmk_core/protocol/usb_hid', 'quantum/template', 'platforms/chibios']
```

Comment:
This wont really be required till `core_dirs` includes `platforms`

Reply:
Added.

Reply:
In which case, we need to also update the travis logic at <https://github.com/qmk/qmk_firmware/blob/master/util/travis_compiled_push.sh#L16> to keep it in step.